### ISSUE TITLE
sensenet Events basic infrastructure

### ISSUE THREAD

**Issue by tusmester** [SenseNet/sensenet#1266]
- [x] implement the basic Events infrastructure, V1
- [x] **processors** can be registered
- [x] events are fired
- [x] all registered processors are called

### Out of scope
- audit log and node observers are not wired in
- events are not saved, fired only once in one app domain
